tests/std/tests/P0645R10_text_formatting_parsing/test.cpp

```diff
@@ -30,17 +29,17 @@ struct choose_literal<wchar_t> {
 
 #define TYPED_LITERAL(CharT, Literal) (choose_literal<CharT>::choose(Literal, L##Literal))
 
-template <typename _CharT>
+template <typename CharT>
 struct testing_callbacks {
     _Align expected_alignment = _Align::_None;
-    basic_string_view<_CharT> expected_fill;
+    basic_string_view<CharT> expected_fill;
     int expected_width               = -1;
     int expected_dynamic_width       = -1;
     bool expected_auto_dynamic_width = false;
     constexpr void _On_align(_Align _Aln) {

```

Comment:
`_Aln` here (and `_Sv` on 42) are unnecessarily `_Ugly`. (Fine to merge these as is and clean up later if there are no other changes necessary.)